src/StructuredLogger/ObjectModel/Project.cs

```diff
@@ -193,5 +193,42 @@ public bool IsLowRelevance
 
         public IReadOnlyList<string> EntryTargets { get; set; } = Array.Empty<string>();
         public IDictionary<string, string> GlobalProperties { get; set; } = ImmutableDictionary<string, string>.Empty;
+
+        public override string ToolTip
+        {
+            get
+            {
+                var sb = new StringBuilder();
+
+                sb.AppendLine(ProjectFile);
+
+                if (EntryTargets != null)
+                {
+                    sb.AppendLine();
+                    sb.AppendLine("Targets:");
+                    sb.AppendLine();
+                    foreach (var target in EntryTargets.OrderBy(target => target, StringComparer.InvariantCultureIgnoreCase))

```

Comment:
I ordered these alphabetically which makes it easier to read, but I fear it might hide information about target ordering. Should this sorting be removed?

Reply:
Shall we have a folder under the Project node for `Targets`? Just like the folder for Global Properties? Preserve the original order, and the user can sort if they want (any node's children can be sorted from the right-click context menu).

I'm torn on whether to sort the targets in the tooltip or not. Keep them sorted for now?